doc/source/getting_started.rst

```diff
@@ -0,0 +1,400 @@
+    arr.ndim
+
+    # array dimensions
+    arr.shape
+
+    # number of elements
+    arr.size
+
+    # size in memory
+    arr.memory_used
+
+    # type of the data of the array
+    arr.dtype
+
+Special arrays can be generated through dedicated functions:
+
+* :py:func:`zeros` : fills an array with 0
+* :py:func:`ones` : fills an array with 1
+* :py:func:`full` : fills an array with a given
+* :py:func:`eye` : identity matrix
+* :py:func:`ndrange` : fills an array with increasing numbers
+* :py:func:`ndtest` : same as ndrange but with axes generated automatically (for testing)
+
+.. ipython:: python
+
+   zeros([age_category, sex])
+
+   ndtest((3, 3))
+
+Save/Load an array
+------------------
+
+The LArray library offers many I/O functions to read and write arrays in various formats
+(CSV, Excel, HDF5, pickle). For example, to save an array in a CSV file, call the method
+:py:meth:`~LArray.to_csv`:
+
+.. ipython:: python
+
+    arr_3D = ndtest((2, 2, 2))
+    arr_3D
+
+    arr_3D.to_csv('arr_3D.csv')
+
+Content of 'arr_3D.csv' file is ::
+
+  a,b\c,c0,c1
+  a0,b0,0,1
+  a0,b1,2,3
+  a1,b0,4,5
+  a1,b1,6,7
+
+.. note::
+   In CSV or Excel files, the last dimension is horizontal and the names of the
+   two last dimensions are separated by a \\.
+
+To load a saved array, call the function :py:meth:`read_csv`:
+
+.. ipython:: python
+
+    arr_3D = read_csv('arr_3D.csv')
+    arr_3D
+
+Other functions are described :ref:`here <api-IO>`.
+
+Indexing
+--------
+
+You can select an element or a subset of an array by labels and/or positions.
+
+Using labels
+^^^^^^^^^^^^
+
+To select an element or a subset of an array using labels, use brackets [ ].
+Let’s start by selecting a single element:
+
+.. ipython:: python
+
+    arr = ndtest((4, 4))
+    arr
+
+    arr['a0', 'b1']
+
+    # labels can be given in arbitrary order
+    arr['b1', 'a0']
+
+Let's continue with subsets:
+
+.. ipython:: python
+
+    # select subset along one axis
+    arr['a0']
+
+    arr['b0']
+
+    # labels associated with the same axis must be given as a list
+    arr['a0', ['b1', 'b3']]
+
+    # or separated by a comma in the same string
+    arr['a0', 'b1,b3']
+
+.. warning::
+
+    Selecting by labels as above only works as long as there is no ambiguity.
+    When several axes have common labels and one does not specify explicitly
+    on which axis to work, it fails with an error
+    (ValueError: ... is ambiguous (valid in a, b)).
+    Specifying the axis can be done using the special notation ``x.axis_name``.
+
+.. ipython:: python
+
+    arr2 = ndrange('a=label0,label1; b=label1,label2')
+    arr2
+
+    arr2['label0', x.b['label1']]
+
+You can also define slices (defined by 'start:stop').
+A slice will select all labels between `start` and `stop`.
+Arguments `start`, `stop` of a slice are optional.
+By default, `start` represents the first label of an axis, `stop` the last one.
+
+.. ipython:: python
+
+    # 'b1:b3' is a shortcut for 'b1,b2,b3'
```

Comment:
I prefer to choose the shortcut string syntax to be consistent with the way I present how to build axes.